Packs/Box/Integrations/BoxEventsCollector/BoxEventsCollector.yml

```diff
@@ -25,12 +25,20 @@ configuration:
   name: page_size
   required: false
   type: 0
-- display: Verify SSL Certificate
-  hidden: false
-  name: verify
+- defaultvalue: box
+  display: Vendor Name
+  name: vendor_name
   required: false
-  defaultvalue: 'true'
+  type: 0
+- defaultvalue: box
+  display: Vendor Name
```

Comment:
Shouldn't it be Product name ?

Reply:
you tell me

Reply:
You have vendor name twice in the yml file, the display is vendor name but the value is product_name

`- defaultvalue: box
  display: Vendor Name
  name: vendor_name
  required: false
  type: 0
- defaultvalue: box
  display: Vendor Name
  name: product_name
  required: false
  type: 0`
@jochman